students/kimjy/westagram/users/views.py

```diff
@@ -31,20 +33,26 @@ def post(self, request):
             if User.objects.filter(email_address=email).exists():
                 raise IntegrityError("EMAIL_DUPLICATED_ERROR")
             
+            hashed_password = bcrypt.hashpw(
+                                      password.encode('utf-8'),
+                                      bcrypt.gensalt()
+                    )
```

Comment:
공백을 너무 많이 주셨습니다. 줄바꿈 후에는 4칸만 주세요
```suggestion
            hashed_password = bcrypt.hashpw(
                password.encode('utf-8'),
                bcrypt.gensalt()
            )
```

Reply:
코멘트 감사합니다.
말씀하신 내용 반영하여 수정하였습니다.